src/orcania.c

```diff
@@ -38,6 +38,13 @@
 #define strcasecmp _stricmp
 #endif
 
+#ifdef _MSC_VER
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
+#else
+#include <unistd.h>
```

Comment:
I'd prefer to have `#include <unistd.h>` not here if it's not required

Reply:
I think I mistakenly copy pasted this from `base64url.c`.